hawkular-inventory-json-helper/src/main/java/org/hawkular/inventory/json/mixins/DataEntityMixin.java

```diff
@@ -19,20 +19,26 @@
 import java.util.Map;
 
 import org.hawkular.inventory.api.model.CanonicalPath;
+import org.hawkular.inventory.api.model.DataEntity;
 import org.hawkular.inventory.api.model.StructuredData;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 /**
  * @author Lukas Krejci
  * @since 0.3.0
  */
-public class DataEntityMixin {
+public abstract class DataEntityMixin {
 
     @JsonCreator
     public DataEntityMixin(@JsonProperty("path") CanonicalPath path, @JsonProperty("value") StructuredData value,
             @JsonProperty("properties") Map<String, Object> properties) {
     }
 
+    @JsonIgnore abstract String getId();
```

Comment:
So we are only left with `path` in JSON right? I think this makes sense for the new entities we add. I left the `tenantId` and all the other guys in the json ouput because I think someone depended on those attributes being there but I think we should remove them going forward because they are duplicate information.